src/metabase/actions.clj

```diff
@@ -1,17 +1,17 @@
 (ns metabase.actions
   "Code related to the new writeback Actions."
-  (:require
-   [clojure.spec.alpha :as s]
-   [metabase.api.common :as api]
-   [metabase.driver :as driver]
-   [metabase.mbql.normalize :as mbql.normalize]
-   [metabase.mbql.schema :as mbql.s]
-   [metabase.mbql.util :as mbql.u]
-   [metabase.models.database :refer [Database]]
-   [metabase.models.setting :as setting]
-   [metabase.util :as u]
-   [metabase.util.i18n :as i18n]
-   [schema.core :as schema]))
+  (:require [clojure.spec.alpha :as s]
+            [medley.core :as m]
+            [metabase.api.common :as api]
+            [metabase.driver :as driver]
+            [metabase.mbql.normalize :as mbql.normalize]
+            [metabase.mbql.schema :as mbql.s]
+            [metabase.mbql.util :as mbql.u]
+            [metabase.models.database :refer [Database]]
+            [metabase.models.setting :as setting]
+            [metabase.util :as u]
+            [metabase.util.i18n :as i18n]
+            [schema.core :as schema]))
```

Comment:
There are two different conventions for this 

```clj
(:require <first-dep>
          ...)
```

or 

```clj
(:require
 <first-dep>
  ...)
```

Different tools seems to automatically format things one way or the other; our codebase has a mix of both.

I don't really care one way or the other, but I do think we should avoid introducing needless churn and inflated diffs by switching the style up in existing files. It's a merge conflict waiting to happen.

We should stick to the existing style in existing namespaces. 

For new namespaces, use whichever style your editor picks for you. But for existing namespaces, it only takes a few seconds to go add a newline or delete it and then reindent the form.